pkgs/development/libraries/drogon/no_cmake_test.patch

```diff
@@ -0,0 +1,42 @@
+diff --git a/test.sh b/test.sh
+index e5da08e..ba620cc 100755
+--- a/test.sh
++++ b/test.sh
+@@ -157,37 +157,6 @@ cd ../views
+ 
+ echo "Hello, world!" >>hello.csp
+ 
+-cd ../build
+-if [ $os = "windows" ]; then
+-  conan install $src_dir -s compiler="Visual Studio" -s compiler.version=16 -sbuild_type=Debug -g cmake_paths
+-  cmake_gen="$cmake_gen -DCMAKE_TOOLCHAIN_FILE=conan_paths.cmake -DCMAKE_INSTALL_PREFIX=$src_dir/install"
+-fi
+-cmake .. $cmake_gen
+-
+-if [ $? -ne 0 ]; then
+-    echo "Error in testing"
+-    exit -1
+-fi
+-
+-cmake --build . -- $make_flags
+-
+-if [ $? -ne 0 ]; then
+-    echo "Error in testing"
+-    exit -1
+-fi
+-
+-if [ $os = "linux" ]; then
+-  if [ ! -f "drogon_test" ]; then
+-      echo "Failed to build drogon_test"
+-      exit -1
+-  fi
+-else
+-  if [ ! -f "Debug\drogon_test.exe" ]; then
+-      echo "Failed to build drogon_test"
+-      exit -1
+-  fi
+-fi
+-
+ cd ../../
+ rm -rf drogon_test
```

Comment:
I feel this makes the file irrelevant...

Why keep it around?

Reply:
How does this patch make `test.sh` irrelevant?  It still does integration testing and it runs `drogon_ctl` a few times with different arguments to make sure it works right.

Reply:
do you mind commenting  (in the nix expression) the intent of the patch?

Reply:
No problem, do you find these more helpful?